docs/source/release_notes/lts/52_0.rst

```diff
@@ -0,0 +1,162 @@
+- Users can now choose to keep the complete set of files, including
+  temporary ones, created during an Avocado job run by using the
+  ``--keep-tmp`` option (e.g. to keep those files for ``rr``).
+- Tests running with the external runner (``--external-runner``) feature
+  will now have access to the extended behavior for SIMPLE tests, such
+  as being able to exit a test with the WARNING status.
+- Encoding support was improved and now Avocado should safely treat
+  localized test-names.
+- Test writers now have access to a test temporary directory that will
+  last not only for the duration of the test, but for the duration of
+  the whole job execution to allow sharing state/exchanging data between
+  tests.
+- When not disabled, "results.tap" is by default generated in results
+  directory.
+- Avocado is now available on Fedora standard repository. The package
+  name is ``python2-avocado``. The optional plugins and examples
+  packages are also available. Run ``dnf search avocado`` to list them
+  all.
+- Optional plugins and examples packages are also Avocado on PyPI.
+- Avocado test writers can now use a family of decorators, namely
+  ``avocado.skip()``, ``avocado.skipIf()`` and ``avocado.skipUnless()``
+  to skip the execution of tests.
+- ``self.skip()`` is under deprecation process and users should use the
+  skip decorators instead.
+- Sysinfo collection based on command execution now allows a timeout to
+  be set.
+- The non-local runner plugins, the html plugin and the yaml_to_mux
+  plugin are now distributed in separate packages.
+- Users sending SIGTERM to Avocado can now expect it to be properly handled.
+- Support for new type of test status, ``CANCEL``, and of course the
+  mechanisms to set a test with this status.
+- ``avocado.TestFail``, ``avocado.TestError`` and ``avocado.TestCancel``
+  are now public Avocado Test APIs, available from the main avocado
+  namespace.
+- Introduction of the robot plugin, which allows Robot Framework tests
+  to be listed and executed natively within Avocado.
+- A brand new ResultsDB optional plugin.
+- Listing of supported loaders (``--loaders \?``) was refined.
+- Variant-IDs generated by yaml_to_mux plugin now include leaf node
+  names to make them more meaningful, making easier to skim through
+  the results.
+- `yaml_to_mux` now supports internal filters defined inside the YAML
+  file expanding the filtering capabilities even further.
+- Avocado now supports resuming jobs that were interrupted.
+- The HTML report now presents the test ID and variant ID in separate
+  columns, allowing users to also sort and filter results based on those
+  specific fields.
+- The HTML report will now show the test parameters used in a test when
+  the user hovers the cursor over the test name.
+- Avocado now reports the total job execution time on the UI, instead of
+  just the tests execution time.
+- The Avocado multiplex command has been renamed to variants.
```

Comment:
Well for the purpose of "new features" I'd probably say `New "avocado variants" has been added which supersedes the "avocado multiplex" command.`